04_health/neonatal_health_code_2022.do

```diff
@@ -0,0 +1,701 @@
+************************************************************************
+* Ancestor program: $gitfolder/neonatal_health_code_2018.do 				   
+* Original data: all_births_by_county.txt, lbw_births_by_county.txt, nomiss_bw_by_county.txt available in $gitfolder/04_health/data      				   
+* Description: Program to create gates mobility metrics on neonatal health  
+* Authors: Emily M. Johnston, Julia Long 												   
+* Date: September 2, 2022	
+* (1)  download data from CDC WONDER											
+* (2)  import, clean, and merge CDC WONDER files					   
+* (3)  intermediate file data cleaning										   
+* (4)  use crosswalk to add missing counties to data file		
+* (5)  assign "unidentified county" values to counties with missing values
+* (6)  create neonatal health share lowbirthweight metric
+* (7) assess data quality
+* (8) construct 95 percent confidence intervals
+* (9) final file cleaning and export to csv file	
+* more steps that start with taking the final county file
+	*sum the numerators/denoms 
+	*use the same confidence interval formula. 
+****************************************************************************
+
+
+/*
+
+Description: Program to Update Gates Mobility Metrics on Neonatal Health
+Author: Jay Carter
+Date: Feb 21, 2024
+
+Data from CDC Wonder - Instructions in Readme
+
+*/
+clear all 
+pause on 
+
+global gitfolder "C:\Users\jcarter\Documents\git_repos\mobility-from-poverty\"	// update path as necessary to the local mobility metrics repository folder
+global health "${gitfolder}04_health\"
+global health_data "${health}data\"
```

Comment:
This directory is not available under 04_health so the code stops running when it tries to pull raw data from that folder